Gemfile

```diff
@@ -48,6 +50,7 @@ group :development, :test do
 
   # Use pry for rails console
   gem 'pry-rails'
+  gem 'binding-of-caller'
```

Comment:
This did not work for me - I believe the gem is `binding_of_caller`, not `binding-of-caller`. This probably means that you made this change to your Gemfile and then never ran `bundle`.